packages/cli/test/commands/upgrade.test.ts

```diff
@@ -51,44 +53,66 @@ describe('upgrade:command', async () => {
 
     const report: Report = readJSONSync(reportFile);
 
-    expect(report).toHaveProperty('summary');
-
-    expect(report.summary.projectName).toBe('@rehearsal/cli');
-    expect(report.summary.tsVersion).toBe(latestPublishedTSVersion);
-    expect(report.summary.uniqueErrors).toBe(3);
-    expect(report.summary.totalErrors).toBe(25);
-    expect(report.summary.totalErrorsList).toStrictEqual({
-      '2322': 1,
-      '2616': 3,
-      '6133': 21, 
-    });
-    expect(report.summary.fixedErrors).toBe(0);
-    expect(report.summary.fixedErrorsList).toStrictEqual({
-      '2322': 0,
-      '2616': 0,
-      '6133': 0,
-    });
-    expect(report.summary.files).toBe(3);
-    expect(report.summary.filesList).toStrictEqual([
-      '/foo/foo.ts',
-      '/foo_2/foo_2a.ts',
-      '/foo_2/foo_2b.ts',
+    expect(report).toMatchSnapshot();
+  });
+});
+
+describe('upgrade:command typescript@rc', async () => {
```

Comment:
```
describe.each(['rc', 'latest', 'beta'])('upgrade:command typescript@%i', async (buildTag) => {
   ...
}
```

or

```
describe('upgrade:command typescript', async () => {
   test.each(['rc', 'latest', 'beta'])('%1 build`, async (buildTag) => {
      ...
   })
}
```

Reply:
i intentionally didn't implement this within a loop so we could have the option to have different checks and conditions for each build. eg. with `rc` we dont want codemods to run and only report etc.